vmm/src/cpu.rs

```diff
@@ -920,15 +920,13 @@ impl CpuManager {
                                 // to be executed.
 
                                 #[cfg(feature = "kvm")]
```

Comment:
Do we still need this compiler flag since we are doing a runtime check?

Reply:
Yes, because the enum member is build conditional.

Reply:
Yes, these feature gates are needed to have kvm-only or mshv-only binaries.

Reply:
Also, this piece of code is purely for kvm and we don't want to build it on MSHV.

Reply:
> Yes, these feature gates are needed to have kvm-only or mshv-only binaries.

But we do support combined binaries these days and for that binary, that compiler check is kind of obsolete since this will always return true. I mean if we want to support combined binary and avoid such kind of bugs in future we might be better off relying on the runtime check instead of compile time check. What do you think about it?